alma/alma-installments-prestashop
# feat: Implement PHP client endpoints into Prestashop

### Reason for change



[Linear task](https://linear.app/almapay/issue/ECOM-2127/[-ps]-implement-php-client-endpoints-into-prestashop)

### Code changes


## First part : CMS is able to send the endpoint_url to backend
We create a function in clientHelper to send url for gather cms data
We create a function in apiHelper to send url for gather cms data
We use the function to send url for gather cms data after the form saved

## Second part : CMS is able to return cms_collected_data to backend
*Blocked by the PR front controller CMS Data*

### How to test

_As a reviewer, you are encouraged to test the PR locally._


First part : Save the config form and see if the url of front controller for gather CMS data is send
Second part : Execute the url executed by Alma and see if the CMS data are send


### Checklist for authors and reviewers




### Non applicable
